### PR TITLE
Button group

### PR DESCRIPTION
**Pull request by tyndria**
http://dev.wir.by/en/topic/authors/tag/karatkievich/

### CODE REVIEW

**Comment by tyndria** [comment]
[topics pages markup & frontend](https://trello.com/c/Jgd2T6A9/365-topics-pages-markup-frontend)

File: components/common/ButtonGroup/ButtonGroup.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import './ButtonGroup.scss';
```
Comment (Drapegnik) [inline components/common/ButtonGroup/ButtonGroup.js]: for consistency

```suggestion
import './button-group.scss';
```

File: components/social/ShareButtons.js [hunk 2]
```diff
@@ -3,6 +3,8 @@ import './social-buttons.scss';
 import React from 'react';
 
 import Icon from 'components/common/ui/Icon';
+import ButtonGroup from 'components/common/ButtonGroup/ButtonGroup';
```
Comment (Drapegnik) [inline components/social/ShareButtons.js]: lets store it just like 
```suggestion
import ButtonGroup from 'components/common/ButtonGroup';
```

File: pages/tag.js [hunk 3]
```diff
@@ -23,6 +26,13 @@ const CARD_SIZE = {
   [MOBILE]: 'square-s',
 };
 
+const buttons = [
```
Comment (Drapegnik) [inline pages/tag.js]:
```suggestion
const BUTTONS = [
```

File: pages/tag.js [hunk 4]
```diff
@@ -23,6 +26,13 @@ const CARD_SIZE = {
...
+  { textId: 'common.load15More' },
```
Comment (Drapegnik) [inline pages/tag.js]:
```suggestion
  { id: 'load15More' },
```

File: pages/tag.js [hunk 5]
```diff
@@ -85,6 +95,13 @@ class TagPage extends Component {
             </>
           )}
         </ScreenContext.Consumer>
+        <ButtonGroup>
+          {buttons.map(({ textId }) => (
+            <Button key={textId}>
+              <Text id={textId} />
```
Comment (Drapegnik) [inline pages/tag.js]:
```suggestion
              <Text id={`common.${id}`} />
```

**Review by Drapegnik — CHANGES_REQUESTED** [review]

**Review by Drapegnik — APPROVED** [review]

**Pull request merged**